R/src.translate.env.src.presto.R

```diff
@@ -14,6 +18,9 @@ src_translate_env.src_presto <- function(x) {
         sql_type <- toupper(dbDataType(Presto(), type))
         dplyr::build_sql('CAST(', column, ' AS ', dplyr::ident(sql_type), ')')
       },
+      `[` = .subscript,
+      `[[` = .subscript,
+      length = dplyr::sql_prefix("cardinality"),
```

Comment:
I'm not sure we can have this because length is already a function to get the length of a string (https://prestodb.io/docs/current/functions/string.html). You should still be able to use `cardinatlity(array_col)` because dplyr knows it is a remote source and doesn't try and evaluate `cardinality` in the local scope.